### PR TITLE
confirmed sessions is working 

### PR DESCRIPTION
**Pull request by bjorkypie**
confirmed sessions works with postman and mongo compass
added isAdmin to cookie
added session secret to env example
changed auth route endpoint in apiRoutes.js
removed unused 'new' variable in Post.js model

Are we on the right track for adding sessions?
Do we need some sort of middleware checking that the cookie matches the session stored in Mongo?

### CODE REVIEW

File: packages/server/routes/authRoutes.js [hunk 1]
```diff
@@ -4,16 +4,26 @@ const bcrypt = require("bcrypt");
 const { checkAdmin } = require("../middleware/auth");
 
 router
+  .get("/getSession", async (req, res) => {
+    res.json({ session: req.session });
+  })
   .post("/login", async (req, res) => {
-    console.log(req.url, req.method, req.body);
+    // console.log(req.url, req.method, req.body);
+
```
Comment (indifferentghost) [inline packages/server/routes/authRoutes.js:12]: Let's clean this up
```suggestion
```

File: packages/server/routes/authRoutes.js [hunk 2]
```diff
@@ -22,26 +32,34 @@ router
     res.status(401).json({ message: "Invalid credentials" });
   })
   .post("/admin/create-user", checkAdmin, async (req, res) => {
-    console.log(req.url, req.method, req.body);
+    // console.log(req.url, req.method, req.body);
+
```
Comment (indifferentghost) [inline packages/server/routes/authRoutes.js:36]: Let's continue to clean this up:
```suggestion
```

File: packages/server/routes/authRoutes.js [hunk 3]
```diff
@@ -22,26 +32,34 @@ router
...
       email,
       username,
       password,
       isAdmin: true,
     });
+
     try {
       const _response = await user.save();
+
       console.log(_response);
```
Comment (indifferentghost) [inline packages/server/routes/authRoutes.js:49]: Let's clean this up too
```suggestion
```

File: packages/server/routes/authRoutes.js [hunk 4]
```diff
@@ -22,26 +32,34 @@ router
...
     } catch (err) {
       console.log(err);
+
       res.status(422).json({ message: err.message });
     }
   })
   .put("/admin/update-password/:id", checkAdmin, async (req, res) => {
-    console.log(req.url, req.method, req.body, req.params);
+    // console.log(req.url, req.method, req.body, req.params);
+
```
Comment (indifferentghost) [inline packages/server/routes/authRoutes.js:60]: continued clean up
```suggestion
```

File: packages/server/server.js [hunk 5]
```diff
@@ -3,6 +3,7 @@ const session = require("express-session");
 const MongoStore = require("connect-mongo");
 const apiRoutes = require("./routes/apiRoutes");
 const path = require("path");
+require("dotenv").config();
```
Comment (indifferentghost) [inline packages/server/server.js:6]: Nice catch, weird that it wasn't here.

You might want to move this to the top

**Review by indifferentghost — CHANGES_REQUESTED** [review]
Small changes suggested, mostly cleanup looks good

**Review by relspeedwagon — APPROVED** [review]

**Review by indifferentghost — APPROVED** [review]

**Pull request merged**